playbooks/azure/openshift-cluster/launch.yml

```diff
@@ -19,8 +19,8 @@
       url: "{{ item }}"
       dest: "{{ tmp.path }}/"
     with_list:
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/acs-engine"
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/openshift.json"
+    - "http://acs-engine-build-acs-engine.svc.ci.openshift.org/acs-engine"
```

Comment:
is this change meant to be here?

Reply:
Will be dropped once I see a successful test run.

https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_extended_conformance_azure_39_test/1/console